packages/framework/src/Component/Normalizer/Normalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Normalizer;
+
+use Symfony\Component\Serializer\Normalizer\ArrayDenormalizer;
+use Symfony\Component\Serializer\Normalizer\GetSetMethodNormalizer;
+use Symfony\Component\Serializer\Serializer as SymfonySerializer;
+
+class Normalizer
```

Comment:
why the name `Normalizer`? It seems that symfony is using term `Denormalizer`

Reply:
normalizer removed completely :wink: